src/ng/directive/form.js

```diff
@@ -485,7 +485,7 @@ var formDirectiveFactory = function(isNgForm) {
             var controller = ctrls[0];
 
             // if `action` attr is not present on the form, prevent the default action (submission)
-            if (!('action' in attr)) {
+            if (!('action' in attr) && attr['method'] !== 'dialog') {
```

Comment:
According to the spec, if method is dialog then it closes the dialog and stops further processing. The action attribute should be ignored by the browser in that case.